### PR TITLE
Added DNS_provier_propagation_seconds to request-cert

### PR DESCRIPTION
**Pull request by Slugger70**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,17 @@ Certbot Git repository options. To install from source, set `certbot_install_fro
...
 The directory inside which Certbot will be cloned.
 
+### DNS providers
+
+If you use a DNS provider such as Cloudflare or similar, the following variables are available:
+
+    certbot_dns_provider: cloudflare
+    certbot_dns_credentials:
+        api_token: your_dns_provider_api_token
+    dns_provider_propagation_seconds: 60
```
Comment (natefoo) [inline README.md]:
```suggestion
    certbot_dns_provider_propagation_seconds: 60
```

File: tasks/request-cert.yml [hunk 2]
```diff
@@ -38,6 +38,9 @@
             {% if certbot_dns_credentials is defined %}
             --dns-{{ certbot_dns_provider }}-credentials {{ certbot_dns_credentials_file }}
             {% endif %}
+            {% if dns_provider_propagation_seconds is defined %}
+            --dns-{{ certbot_dns_provider }}-propagation-seconds {{ dns_provider_propagation_seconds }}
```
Comment (natefoo) [inline tasks/request-cert.yml]:
```suggestion
            --dns-{{ certbot_dns_provider }}-propagation-seconds {{ certbot_dns_provider_propagation_seconds }}
```

File: README.md [hunk 3]
```diff
@@ -34,6 +34,17 @@ Certbot Git repository options. To install from source, set `certbot_install_fro
...
+
+See `certbot --help all` for details
```
Comment (natefoo) [inline README.md]:
```suggestion
Valid options for `certbot_dns_credentials` vary by DNS provider. See `certbot --help all` for details.
```

**Review by natefoo — COMMENTED** [review]
Thanks for documenting my garbage!

File: tasks/request-cert.yml [hunk 4]
```diff
@@ -38,6 +38,9 @@
             {% if certbot_dns_credentials is defined %}
             --dns-{{ certbot_dns_provider }}-credentials {{ certbot_dns_credentials_file }}
             {% endif %}
+            {% if dns_provider_propagation_seconds is defined %}
```
Comment (hexylena) [inline tasks/request-cert.yml]:
```suggestion
            {% if certbot_dns_provider_propagation_seconds is defined %}
```

**Review by hexylena — APPROVED** [review]

**Pull request merged**